server/index.js

```diff
@@ -38,11 +38,13 @@ airbnbSchema.plugin(AutoIncrement, { inc_field: 'id' });
 
 const Airbnbs = mongoose.model('Airbnbs', airbnbSchema);
 
-app.get('/api/moreplacestostay', (req, res) => { // Make different API call urls + have them return 12 docs
-  // changed to avoid getting ALL DATA
-  Airbnbs.find({})
-    .limit(12)
-    .exec((err, data) => {
+// Make different API call urls + have them return 12 docs
+app.get('/api/moreplacestostay', (req, res) => {
+  // Use random input to avoid always getting first few data
+  // add search condition to avoid getting ALL DATA
```

Comment:
If this line explains another line, it should be right above that line. I couldn't say that this is what is being done on line 45.